bullet/src/Base.hh

```diff
@@ -56,7 +56,8 @@ struct ModelInfo
 {
   std::string name;
   Identity world;
-  std::vector<std::size_t> links = {};
+  bool fixed;
```

Comment:
If the models are entities on its own, shouldn't be a boolean like this be on top of the links instead? Can models be composed of fixed links + not fixed ones?